questdb/results/c6a.metal.json

```diff
@@ -1,57 +1,57 @@
 {
     "system": "QuestDB",
-    "date": "2024-11-25",
+    "date": "2025-08-16",
     "machine": "c6a.metal",
     "cluster_size": 1,
     "proprietary": "no",
     "tuned": "no",
     "comment": "Uses multi-threaded COPY SQL for data load.",
     "tags": ["Java", "time-series"],
-    "load_time": 157,
-    "data_size": 72847067281,
+    "load_time": 231,
+    "data_size": 72847034489,
     "result": [
-        [0.032,0.000,0.000],
-        [0.405,0.017,0.018],
-        [0.196,0.008,0.012],
-        [2.848,0.009,0.009],
-        [3.653,0.868,0.281],
-        [10.039,0.128,0.556],
-        [2.059,0.017,0.026],
-        [0.183,0.104,0.023],
-        [2.738,2.078,1.946],
-        [3.500,3.051,1.967],
-        [0.622,0.120,0.114],
-        [0.272,0.117,0.101],
-        [0.570,0.222,0.206],
-        [0.538,0.266,0.274],
-        [1.379,0.202,0.198],
-        [1.770,0.455,0.420],
-        [1.353,0.776,0.771],
-        [1.068,0.645,0.631],
-        [4.058,2.326,2.351],
-        [0.010,0.009,0.012],
-        [42.267,0.110,0.103],
-        [0.432,0.104,0.111],
-        [41.872,0.096,0.096],
-        [7.348,0.027,0.024],
-        [0.142,0.006,0.006],
-        [0.647,0.564,0.619],
-        [0.007,0.005,0.004],
-        [2.354,0.160,0.203],
-        [30.692,1.959,1.994],
-        [0.013,0.005,0.005],
-        [1.328,0.146,0.148],
-        [2.134,0.209,0.201],
-        [2.994,2.183,2.232],
-        [0.916,0.783,0.790],
-        [1.370,1.257,1.220],
-        [0.692,0.700,0.701],
-        [0.130,0.076,0.084],
-        [0.069,0.046,0.052],
-        [0.103,0.015,0.019],
-        [0.299,0.206,0.194],
-        [3.686,0.025,0.025],
-        [1.822,0.018,0.023],
-        [0.025,0.006,0.005]
+        [0.103846,0.000413741,0.000331868],
```

Comment:
Would you please change the scripts such that the result files contain only three digits after the dot? (e.g. `0.389` instead of `0.389209`)

Reply:
Rounded the numbers in c218eb67b51fa968c54d6727f60495ce6713568b